R/utils-summary.R

```diff
@@ -0,0 +1,21 @@
+#' Count number of data points since specified date
+#'
+#' `count_since` takes a `billionaire::load_billion_data()` dataframe and
+#' count the number of data points exists for each indicator in `df` since the
+#' specified `year`.
+#'
+#' @inheritParams transform_hpop_data
+#' @param year numeric
+#'
+#' @return data frame
+
+count_since <- function(df, year) {
+  billionaiRe:::assert_mart_columns(df)
```

Comment:
No need to refer to functions internal to the package NAMESPACE with `billionaiRe:::`

Reply:
My bad, was an artifact from testing